src/Microsoft.Tye.Proxy/Program.cs

```diff
@@ -19,7 +19,7 @@ public class Program
         static async Task Main(string[] args)
         {
             var serviceName = Environment.GetEnvironmentVariable("APP_INSTANCE");
```

Comment:
Do we need similar check for "APP_INSTANCE"?

Reply:
No, because we don't rely on it in the same way we do the other (i.e. it's logged but not actually used).